docs/book/src/topics/data-disks.md

```diff
@@ -31,6 +31,22 @@ To check all available vm-sizes in a given region which supports availability zo
 ```bash
 az vm list-skus -l <location> -z -s <VM-size>
 ```
+
+Provided that the chosen region and zone support Ultra disks, Azure Machines objects having Ultra disks specified as Data disks will have their virtual machines created with the `AdditionalCapabilities.UltraSSDEnabled` additional capability set to `true`. This capability can also be manually set on the Azure Machine spec and will override the automatically chosen value (if any).
```

Comment:
Nit, I would expect this to be Azure Machine, unless this is a common pattern in these docs and some nuance I've missed 🤔 
```suggestion
Provided that the chosen region and zone support Ultra disks, Azure Machine objects having Ultra disks specified as Data disks will have their virtual machines created with the `AdditionalCapabilities.UltraSSDEnabled` additional capability set to `true`. This capability can also be manually set on the Azure Machine spec and will override the automatically chosen value (if any).
```

Reply:
Updated, thanks!